jest-integration.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
What is the importance of this file?

Reply:
it defines the configuration jest should use when running integration tests. 
we refer to it when running the tests this way

`"test:integration": "env-cmd -e test jest --config ./jest-integration.json"`

there could be a point when we just want to only run either unit tests  or only integration tests, for some reason or another hence necessary to define configurations for the different types of tests.

Reply:
Thank you for the clarification